lib/server/server.js

```diff
@@ -558,8 +558,9 @@ function execute(port) {
   // gaze watches some specified dirs and triggers a callback when they change.
   gaze(
     [
-      '../docs/**/*', // docs
+      '../' + readMetadata.getDocsPath() + '/**/*', // docs
```

Comment:
This seemed to work ok, right? I didn't test it locally 😄

Reply:
hehe, it is, I tested it :)